.github/workflows/test.yml

```diff
@@ -30,6 +30,8 @@ jobs:
   test:
     needs: pre_job
     runs-on: ${{ matrix.os }}
+    env:
+      TASTY_NUM_THREADS: 1
```

Comment:
this has a different effect from pass `-1` to the test executable?

Reply:
it seems the env var is equivalent to the arg, if docs are correct: https://github.com/UnkindPartition/tasty#runtime

Reply:
It is equivalent, but I don't think it worked

Reply:
but we are already passing `-j1` to tasty via `--test-options` in the workflow, do you suspect the cli arg could have a bug?

Reply:
Oh you are right, I missed that `-j1` is already there

Reply:
it is a shot in the dark but i am trying with `-j1` directly for cabal